rest-api/tools/import_organizations.py

```diff
@@ -202,7 +202,10 @@ def _get_lat_long_for_site(self, address_1, city, state):
         logging.warn('Geocode results failed for %s.', full_address)
         return None, None
     except ValueError as e:
-      logging.exception('Invalid geocode key: %s . error: %s', api_key, e)
+      logging.exception('Invalid geocode key: %s. ERROR: %s', api_key, e)
+      return None, None
+    except IndexError as e:
```

Comment:
Does this still actually get thrown with your above changes for a bad address?

Reply:
It is possible.